src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -665,6 +665,23 @@ public override void VisitEmpty(IEmptyOperation operation)
         public override void VisitReturn(IReturnOperation operation)
```

Comment:
>VisitReturn [](start = 29, length = 11)

Please also adjust ```TestOperationVisitor``` to query the new property (at least it shouldn't crash).

Reply:
I added an assertion, which is slightly stronger than you mention because we can rule out the use of `In`.